Sections/US-States/IA/GPP Extension: IAB Privacy’s Iowa Privacy Technical Specification.md

```diff
@@ -0,0 +1,234 @@
+        </td>
+        <td>N-Bitfield(2,8)</td>
+        <td>Two bits for each Data Activity:<p></p><code>0</code> = Not Applicable, the Controller does not Process the specific category of Sensitive Data<p></p><code>1</code> = Opted Out<p></p><code>2</code> = Did Not Opt Out&nbsp;<span style="color:rgb(36, 41, 47);"><p></p>(1). Opt-Out of the Processing of the Consumer’s Sensitive Data Consisting of Personal Data Revealing Racial or Ethnic Origin.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(2). Opt-Out of the Processing of the Consumer’s Sensitive Data Consisting of Personal Data Revealing Religious Beliefs.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(3). Opt-Out of the Processing of the Consumer’s Sensitive Data Consisting of Personal Data Revealing a Mental or Physical Health Diagnosis.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(4). Opt-Out of the Processing of the Consumer’s Sensitive Data Consisting of Personal Data Revealing Sexual Orientation.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(5). Opt-Out of the Processing of the Consumer’s Sensitive Data Consisting of Personal Data Revealing Citizenship or Citizenship Status.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(6). Opt-Out of the Processing of the Consumer’s Sensitive Data Consisting of Genetic Data that May Be Processed for the Purpose of Uniquely Identifying an Individual.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(7). Opt-Out of the Processing of the Consumer’s Sensitive Data Consisting of Biometric Data that May Be Processed for the Purpose of Uniquely Identifying an Individual.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(8). Opt-Out of the Processing of the Consumer’s Sensitive Data Consisting of Precise Geolocation Data.</span><p></p>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">KnownChildSensitiveDataConsents</span>
+        </td>
+        <td>Int(2)</td>
+        <td>Consent to Process Sensitive Data from a Known Child<p></p><code>0</code> = Not Applicable, the Controller does not
+          <span style="color:rgb(36, 41, 47);">Process Sensitive Data of a known Child</span><p></p><code>1</code> = No Consent<p></p><code>2</code> = Consent&nbsp;</td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">MspaCoveredTransaction</span>
+        </td>
+        <td>Int(2)</td>
+        <td>
+          <span style="color:rgb(36, 41, 47);">Publisher or Advertiser, as applicable, is a signatory to the IAB Multi-State Privacy Agreement (MSPA), as may be amended from time to time, and declares that the transaction is a “Covered Transaction” as defined in the MSPA.&nbsp;</span><p></p><code>1</code> = Yes<p></p><code>2</code> = No</td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">MspaOptOutOptionMode</span>
```

Comment:
@jaredmoscow - this file seems to be missing AdditionalDataProcessingConsent... was that intentional?